coral-schema/src/main/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverter.java

```diff
@@ -91,7 +91,7 @@ public Schema toAvroSchema(String dbName, String tableOrViewName, boolean strict
    *                        the resulting schema. The rule is as follows:
    *                                1. Top level namespace is dbName.tableOrViewName
    *                                2. Nested namespace is parentNamespace.parentFieldName
-   * @param forceLowercase if forceLowercase is set to True, we will return schema with lowercased field names
+   * @param forceLowercase if forceLowercase is set to True, we use lowercase schema for union
```

Comment:
Why do we say `we use lowercase schema for union`?
Looks like the newly added unit test is testing a conversion for schema without union.

Reply:
Actually I find out the issue in this unit test `testUnionForceLowercase`, please check the comment above. sorry I should include this test in the PR description.

Reply:
Yeah I saw your description, I don't think it's related to union in particular. 
The issue you mentioned in the description happened because previously, we only used `forceLowercase` while getting the base table's avro schema [here](https://github.com/linkedin/coral/blob/bcfb470a0bbc6ccf8f409363ebe3bc88d2464d45/coral-schema/src/main/java/com/linkedin/coral/schema/avro/SchemaUtilities.java#L129), so that if we set the alias to be non-lowercased (`Id` in your test case), the derived schema is also non-lowercased given we only set lowercase while getting base table's schema, which will be overwritten to be non-lowercased afterwards while visiting the alias.

Reply:
I see, make sense, I will change this comment to "we will return schema with lowercased field names **(but it won't affect upper cased alias)**", does this works?

Reply:
I think it will affect upper cased alias because it overwrites the derived schema to be lowercased almost as the last step, I think just keeping the existing comment is fine.

Reply:
Sure, sounds good.